src/LibHdb++Cassandra.cpp

```diff
@@ -1119,637 +1286,489 @@ void HdbPPCassandra::extract_and_bind_rw_values(CassStatement* statement,
 					extract_and_bind_encoded(statement,param_index, data_format, data, EXTRACT_SET); break;
 				default:
 				{
-					TangoSys_MemStream os;
-					os << "Attribute " << data->attr_name << " type (" << (int)(data_type) << ")) not supported";
-					cerr << __func__ << ": " << os.str() << endl;
+					ERROR_MSG("Attribute " << data->attr_name << " type (" << (int)(data_type) << ")) not supported" << endl);
+					TRACE_MSG("Leaving");
 					return;
 				}
 			} // switch(data_type)
 		}
 		param_index++;
 	}
+
+	TRACE_MSG("Leaving");
 }
 
-int HdbPPCassandra::find_last_event(const CassUuid & id, string &last_event, const string & fqdn_attr_name)
+bool HdbPPCassandra::find_last_event(const CassUuid & id, string &last_event, const string & fqdn_attr_name)
 {
+	TRACE_MSG("Entering");
 	ostringstream query_str;
-	CassStatement* statement = NULL;
-	CassFuture* future = NULL;
 	last_event="??";
 
 	query_str   << "SELECT " << HISTORY_COL_EVENT
-                << " FROM " << m_keyspace_name << "." << HISTORY_TABLE_NAME
-                << " WHERE " << HISTORY_COL_ID << " = ?"
-                << " ORDER BY " << HISTORY_COL_TIME << " DESC LIMIT 1" << ends;
-
-
-    CassError rc = CASS_OK;
+				<< " FROM " << m_keyspace_name << "." << HISTORY_TABLE_NAME
```

Comment:
I would use spaces instead of tabs here too (please fix if possible similar misalignment issues)...
This is just cosmetic.